tests/50-user-notification.tests

```diff
@@ -0,0 +1,11 @@
+#
+# libseccomp regression test automation data
+#
+# Copyright Cisco Systems 2019
+# Author: Tycho Andersen <tycho@tycho.ws>
+#
+
+test type: live
+
+# Testname	API	Result
+50-user-notification	5	ZERO
```

Comment:
This is a nit, but as a convention, the "live" tests follow the naming format of "<num>-live-<test_name>" where <test_name> uses underscores not hyphens.